cmd/mt-index-cat/main.go

```diff
@@ -157,6 +172,22 @@ func main() {
 	for _, d := range defs {
 		if prefix == "" || strings.HasPrefix(d.Metric, prefix) {
 			if substr == "" || strings.Contains(d.Metric, substr) {
+				if tags != "" {
```

Comment:
unimportant: the above two `if`s could also be inverted and result in `continue` if true, just to reduce the number of nested blocks

Reply:
yep, adjusted. looks better now!